resources/language/Hebrew/strings.po

```diff
@@ -0,0 +1,414 @@
+msgstr "רשימת ניגון"
+
+msgctxt "#30203"
+msgid "Stations"
+msgstr "תחנות"
+
+msgctxt "#30204"
+msgid "Auto Playlists"
+msgstr "רשימת ניגון אוטומטיות"
+
+msgctxt "#30205"
+msgid "Artists"
+msgstr "אמנים"
+
+msgctxt "#30206"
+msgid "Albums"
+msgstr "אלבומים"
+
+msgctxt "#30207"
+msgid "Genres"
+msgstr "סגנונות"
+
+msgctxt "#30208"
+msgid "Search"
+msgstr "חיפוש"
+
+msgctxt "#30209"
+msgid "My Library"
+msgstr "הספרייה שלי"
+
+msgctxt "#30210"
+msgid "Shuffle My Library"
+msgstr "ערבב את הספרייה שלי"
+
+msgctxt "#30211"
+msgid "I'm feeling lucky mix"
+msgstr "מיקס - אני מרגיש בר מזל"
+
+msgctxt "#30212"
+msgid "Composers"
+msgstr "מלחינים"
+
+msgctxt "#30213"
+msgid "Songs"
+msgstr "פזמון"
+
+msgctxt "#30214"
+msgid "Shuffle Albums"
+msgstr "ערבב אלבומים"
+
+msgctxt "#30215"
+msgid "Thumbs up"
+msgstr "לייק"
+
+msgctxt "#30216"
+msgid "Last added"
+msgstr "האחרון שנגמר"
+
+msgctxt "#30217"
+msgid "Free and purchased"
+msgstr "בחינם ושנרכש"
+
+msgctxt "#30218"
+msgid "Most played"
+msgstr "נוגנו הכי הרבה"
+
+msgctxt "#30219"
+msgid "Listen Now"
+msgstr "הקשב עכשיו"
+
+msgctxt "#30220"
+msgid "Top charts"
+msgstr "הכי גבוה במצעדים"
+
+msgctxt "#30221"
+msgid "New Releases"
+msgstr "שיחרורים חדשים"
+
+msgctxt "#30222"
+msgid "Browse stations"
+msgstr "עיון בתחנות"
+
+# ###
+# Context actions
+# ###
+msgctxt "#30301"
+msgid "Play all"
+msgstr "נגן הכל"
+
+msgctxt "#30302"
+msgid "Play all - shuffle"
+msgstr "נגן הכל - ערבב"
+
+msgctxt "#30303"
+msgid "Update playlist"
+msgstr "עדכן רשימת ניגון"
+
+msgctxt "#30304"
+msgid "Update playlists"
+msgstr "עדכן רשימות ניגון"
+
+msgctxt "#30305"
+msgid "Update library"
+msgstr "עדכון הספרייה"
+
+msgctxt "#30306"
+msgid "Add to favourites"
+msgstr "הוסף למעודפים"
+
+msgctxt "#30307"
+msgid "Add to playlist"
+msgstr "הוסף לרשימת ניגון"
+
+msgctxt "#30308"
+msgid "Remove from playlist"
+msgstr "הסר מרשימת ניגון"
+
+msgctxt "#30309"
+msgid "Add to library"
+msgstr "הוסף לספרייה"
+
+msgctxt "#30310"
+msgid "Start Radio"
+msgstr "התחל רדיו"
+
+msgctxt "#30311"
+msgid "Search in Youtube"
+msgstr "חפש ביו-טיוב"
+
+msgctxt "#30312"
+msgid "Play all in Youtube"
+msgstr "נגן הכול ביו-טיוב"
+
+msgctxt "#30313"
+msgid "Play in Youtube"
+msgstr "נגן ביו-טיוב"
+
+msgctxt "#30314"
+msgid "Export"
+msgstr "ייצא"
+
+msgctxt "#30315"
+msgid "Add to queue"
+msgstr "הכנס פריט לתור"
+
+msgctxt "#30316"
+msgid "Create playlist"
+msgstr "צור רשימת ניגון"
+
+msgctxt "#30317"
+msgid "Delete playlist"
+msgstr "מחק רשימת ניגון"
+
+msgctxt "#30318"
+msgid "Delete station"
+msgstr "מחק תחנה"
+
+msgctxt "#30319"
+msgid "Artist top songs"
+msgstr "השירים הכי טובים של האמן"
+
+msgctxt "#30320"
+msgid "Related artists"
+msgstr "אמנים קשורים"
+
+msgctxt "#30321"
+msgid "Play in Youtube - shuffle"
+msgstr "נגן ביו-טיוב - ערבב"
+
+# ###
+# Messages
+# ###
+msgctxt "#30101"
+msgid "Login failed"
+msgstr "הכניסה נכשלה"
+
+msgctxt "#30102"
+msgid "Invalid credentials or connectivity problems"
+msgstr "אישורי כניסה לא תקינים או בעיות קישור"
```

Comment:
I think 'אישורי כניסה' should be 'שם משתמש/סיסמה'

Reply:
OK

Reply:
OK